cluster/gce/container-linux/configure-helper.sh

```diff
@@ -144,7 +144,7 @@ function create-master-auth {
   fi
   local -r known_tokens_csv="${auth_dir}/known_tokens.csv"
   if [[ ! -e "${known_tokens_csv}" ]]; then
-    echo "${KUBE_BEARER_TOKEN},admin,admin" > "${known_tokens_csv}"
+    echo "${KUBE_BEARER_TOKEN},admin,admin,system:masters" > "${known_tokens_csv}"
```

Comment:
hmm... the file existence check above means these won't get re-written on upgrade